torch/csrc/jit/ivalue.h

```diff
@@ -251,10 +270,9 @@ struct IValue {
   }
   std::ostream& formatString(std::ostream& out) const {
     JIT_ASSERT(isString());
-    out << toString();
+    out << toRetainable<ConstantString>()->string();
```

Comment:
Why not `toString`?

Reply:
It gets printed as a pointer if I do it that way... Not entirely sure why